jekyll/_cci2/introduction-to-the-circleci-web-ui.adoc

```diff
@@ -0,0 +1,177 @@
+
+**Notifications**: A tooltip with new features or changes. New notifications are identified with a badge next to the navigation item.
+
+**Status**: The status badge will display the operational health of CircleCI. Clicking on this navigation item will bring you to CircleCI's status page.
+
+**Support**: A tooltip with links to community help forums, CircleCI support, as well as links to CircleCI's feature requests and changelog pages.
+
+At the very bottom of the side navigation (you may have to scroll down), you will see your individual **user icon**. Clicking on this will bring you to your individual **User Settings**.
+
+The sections below describe some of the pages of the web application in more detail.
+
+[#projects]
+== Projects
+
+Projects associated with your organization will appear on the project's page. You will have the option to follow or unfollow projects that are already set up with CircleCI, or you can choose to set up a project with CircleCI. Following or setting up a project will add it to your dashboard. Unfollowing a project will remove it from your dashboard.
+
+Clicking the three dots (meatball menu) in the project row will allow you to open an individual project's configuration file, insights, and settings. If you are following a project, you will also see an option to unfollow the project from this menu. There is also a **Project Settings** button found on the dashboard of individual projects, which you can view by clicking on a project's name in the main dashboard.
+
+image::{{site.baseurl}}/assets/img/docs/web_ui_projects.png[Projects page]
+
+**Configuration File**: This option opens the project's configuration file inside a text editor within CircleCI's web UI. You can make changes to the config file and commit them. At the top of the text editor, you will see the project's branch you are committing the changes to (most often `main`).
+
+image::{{site.baseurl}}/assets/img/docs/web_ui_text_editor.png[Text editor in the web UI]
+
+**Insights**: This is a link to the insight's page, which is detailed <<#insights,below>>.
+
+[#project-settings]
+=== Project Settings
+
+**Overview**: Shows your project ID, as well as links to docs on how to set up certain features available to projects. You may also find the option to **Stop Building** on the overview page. This option halts builds, and will automatically _unfollow_ from all dashboards.
+
+**Triggers**: Allows you to define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met.
+
+**Advanced**: Toggle options on and off for: VCS status updates, build forked pull requests, pass secrets to builds from forked pull requests, only build pull requests, auto-cancel redundant workflows, free and open source, enable dynamic config using setup workflows. More information on these settings can be found on this page, as well as in our documentation.
```

Comment:
Not sure if actually worth adding a link here but we do also have this page: https://circleci.com/docs/2.0/settings/ It's a bit weird though since there's not much content on there specifically. Could probably use a refresh.

Reply:
I've made a ticket to assess the relevance of this page 👍